desktop_app/src/ui/components/Chat/ChatHistory/Messages/AssistantMessage.tsx

```diff
@@ -143,20 +134,6 @@ export default function AssistantMessage({
         {isRegenerating ? <RegenerationSkeleton /> : orderedElements}
       </div>
 
-      {/* Token usage display */}
-      {tokenUsage && tokenUsage.totalTokens && (
-        <div className="mt-2 flex items-center gap-2">
-          <TokenUsageDisplay
-            promptTokens={tokenUsage.promptTokens}
-            completionTokens={tokenUsage.completionTokens}
-            totalTokens={tokenUsage.totalTokens}
-            model={tokenUsage.model}
-            contextWindow={tokenUsage.modelContextWindow}
-            variant="inline"
-          />
-        </div>
-      )}
```

Comment:
I guess this is _never_ currently displayed for assistant messages right now, right?